install/0000_50_cluster-autoscaler-operator_03_rbac.yaml

```diff
@@ -47,6 +47,7 @@ rules:
   - "*"
 - apiGroups:
   - cluster.k8s.io
+  - machine.openshift.io
```

Comment:
Don't we need `machine.openshift.io` here too?

https://github.com/openshift/cluster-autoscaler-operator/blob/39f0bee0f2c0108aa02d383c31f45ec1bf4a5987/install/0000_50_cluster-autoscaler-operator_03_rbac.yaml#L149

Reply:
Oops. Yes.